src/program.ts

```diff
@@ -86,6 +98,11 @@ export class Program extends (EventEmitter as new () => TypedEventEmitter<Events
       this.options.historyFile = path.join(os.homedir(), DEFAULT_HISTORY_FILE)
     }
 
+    // Set default exit handler
+    if (this.options.exit && typeof this.options.exit !== 'function') {
```

Comment:
When `this.options.exit` is set to `true` it doesn't set the default value.
```suggestion
    if (this.options.exit === true || typeof this.options.exit === 'undefined') {
```